Tools/WinMLDashboard/public/convert.py

```diff
@@ -7,6 +7,7 @@ def parse_args():
     parser.add_argument('source', help='source  model')
     parser.add_argument('framework', help='source framework model comes from')
     parser.add_argument('ONNXVersion', help='which ONNX Version to convert to')
+    parser.add_argument('inputNames', help='names of input nodes (optional)')
```

Comment:
The argument isn't optional as-is, maybe make it optional or change the doc to `names of input nodes (or empty string)`

Reply:
Will add required=False

Reply:
it looks like all the options are optional by default, so this should be OK as is?

Reply:
At least destination (the last parameter) should be non-empty I think. You'd need a non-positional argument (like `--inputName`) or having inputNames in the end of the argument list

Reply:
hmmm, I have tested it and it was working as is, are you commenting because you believe this won't work as expected or because it's not a good practice?

Reply:
Where is the model saved to if the destination is empty? I though it was a required parameter

Reply:
When the conversion is done, a file dialog will be opened to let the user choose destination & file name